src/shared/utils/index.js

```diff
@@ -1,3 +1 @@
-export * from './ownerHelpers'
 export * from './provider'
```

Comment:
If it's not too much work, I feel like we can just get rid of this file completely at this point.

Reply:
Can do, I'll remove it in https://github.com/codecov/gazebo/pull/3461 when I'm done cooking that one (its been a mega pain)